doc/usage.rst

```diff
@@ -89,34 +89,79 @@ can be updated using the method update_flags.
 Simple loop
 ===========
 
-Bytecode of ``for x in (1, 2, 3): print(x)``::
-
-    from bytecode import Label, Instr, Bytecode
-
-    loop_start = Label()
-    loop_done = Label()
-    loop_exit = Label()
-    code = Bytecode([Instr('SETUP_LOOP', loop_exit),
-                     Instr('LOAD_CONST', (1, 2, 3)),
-                     Instr('GET_ITER'),
-                     loop_start,
-                         Instr('FOR_ITER', loop_done),
-                         Instr('STORE_NAME', 'x'),
-                         Instr('LOAD_NAME', 'print'),
-                         Instr('LOAD_NAME', 'x'),
-                         Instr('CALL_FUNCTION', 1),
-                         Instr('POP_TOP'),
-                         Instr('JUMP_ABSOLUTE', loop_start),
-                     loop_done,
-                         Instr('POP_BLOCK'),
-                     loop_exit,
-                         Instr('LOAD_CONST', None),
-                         Instr('RETURN_VALUE')])
-
-    # the conversion to Python code object resolve jump targets:
-    # replace abstract labels with concrete offsets
-    code = code.to_code()
-    exec(code)
+Bytecode of ``for x in (1, 2, 3): print(x)``:
+
+.. tabs::
+
+    .. group-tab:: Python < 3.8
```

Comment:
`3.6 <= Python < 3.8`

There was different code for calling a function in <3.6.

Reply:
I never realized the documentation was using Python 3.5 incompatible bytecode. I will fix this.

Reply:
This is what I get using Python 3.5.6 and I do not see any difference apart from the code offset in this case at least.

```python
import dis
>>> def f():
...     for x in (1, 2, 3): print(x)
... 
>>> dis.dis(f)
  2           0 SETUP_LOOP              24 (to 27)
              3 LOAD_CONST               4 ((1, 2, 3))
              6 GET_ITER
        >>    7 FOR_ITER                16 (to 26)
             10 STORE_FAST               0 (x)
             13 LOAD_GLOBAL              0 (print)
             16 LOAD_FAST                0 (x)
             19 CALL_FUNCTION            1 (1 positional, 0 keyword pair)
             22 POP_TOP
             23 JUMP_ABSOLUTE            7
        >>   26 POP_BLOCK
        >>   27 LOAD_CONST               0 (None)
             30 RETURN_VALUE
```

Reply:
ping @serhiy-storchaka

Reply:
You are right. There are different opcodes for calling a function in 3.5 and 3.6, but in this particular case (passing a fixed number of positional arguments) they coincide.

Reply:
Thanks for confirming. I will merge and release since people need the Python 3.8 support and will revisit the extension of the documentation later.